db/migrations/mysql/000031_create_threads.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS Threads (
+    PostId varchar(26) NOT NULL,
+    ReplyCount bigint(20),
+    LastReplyAt bigint(20),
+    Participants text,
```

Comment:
This column is now json. Are we incorporating the latest schema changes from master or doing that later?

Reply:
Actually the whole thing PRs currently pointing to the [MM-31622/schema-migrations](https://github.com/mattermost/mattermost-server/tree/MM-31622/schema-migrations) so this is being updated to that version. Once we cleanup and merge I'm planning to update to the recent changes and merge the feature branch.

Reply:
Sounds good.